public/docs/js/config.js

```diff
@@ -0,0 +1,4 @@
+window.config = {
+  base_url: "http://localhost:9393",
```

Comment:
Will need this to use current URL somehow? Could it use `window.location.origin`?